heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerRunner.java

```diff
@@ -161,6 +162,25 @@ void killTopologyHandler(String topologyName) throws TopologyRuntimeManagementEx
     LOG.fine("Scheduler killed topology successfully.");
   }
 
+  boolean consolePrompt(int oldContainerCount, int newContainerCount) {
+    String fmt =
+        "The present aurora job has %d containers. After update there will be %d containers. ";
```

Comment:
looks like you dont need the space at the end of fmt.

Reply:
removed